matrix_client/api.py

```diff
@@ -263,6 +263,29 @@ def send_notice(self, room_id, text_content):
         }
         return self.send_message_event(room_id, "m.room.message", body)
 
+    def get_room_messages(self, room_id, token, direction="b", limit=10, to=None):
```

Comment:
Nit pick: Could you not make direction default to back? I feel like if people called this, if anything they'd expect it to default to a forwards direction. I think the best solution though would be to explicitly require it.